src/Microsoft.OpenApi/Expressions/RuntimeExpression.cs

```diff
@@ -99,5 +99,11 @@ public bool Equals(RuntimeExpression obj)
         {
             return obj != null && obj.Expression == Expression;
         }
+
+        /// <inheritdoc />
+        public override string ToString()
+        {
+            return Expression;
+        }
```

Comment:
Not sure when we need this?

Reply:
Yes. I think so. It can be used to anywhere if you want the string.

Reply:
I guess it's okay. I was a bit worried about how people interpret what ToString() should output. But I don't think there's other choice on how to output runtime expression as a string.

Reply:
Feels natural to me.